packages/paste-customization/src/utils.ts

```diff
@@ -0,0 +1,19 @@
+import {ThemeShape} from '@twilio-paste/theme';
+
+const cloneDeep = require('lodash/cloneDeep');
+const merge = require('lodash/merge');
+
+interface CreateCustomTheme {
+  (baseTheme: Partial<ThemeShape>, overrides: {}, breakpoints?: string[]): {};
+}
+export const createCustomTheme: CreateCustomTheme = (baseTheme, overrides, customBreakpoints) => {
+  const breakpoints = customBreakpoints || baseTheme.breakpoints;
+  // clone so we don't mutate the default theme
+  // this way you can get multiple custom themes on the same page
+  const clonedBaseTheme = cloneDeep(baseTheme);
+  const mergedTheme = merge(clonedBaseTheme, overrides);
```

Comment:
I think you can do `merge({}, baseTheme, overrides)` to prevent mutation and save on the cloneDeep call

Reply:
Very Nice!